products/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.core.exceptions import MultipleObjectsReturned
+from django.http            import JsonResponse
+from django.views           import View
+
+from .models      import Food, Category, FoodImage, RequiredOption, SelectOption
+
+class FoodView(View):
+    def get(self, request, food_id):
+        try:
+            food             = Food.objects.get(id=food_id)
+
+            result = {
+                'name'             : food.name,
+                'price'            : food.price,
+                'discount'         : food.discount,
+                'discounted_price' : food.discounted_price,
+                'star_score'       : food.star_score,
+                'review_count'     : food.review_count,
+                'detail_image'     : food.detail_image,
+                'create_at'        : food.create_at,
+                'update_at'        : food.update_at,
+                'image'            : [image.image_url for image in FoodImage.objects.filter(food=food_id)],
```

Comment:
이미 위에서 food 객체를 가져온 상태입니다. `Food`와 `FoodImage`는 1:M관계 이므로 food에서 직접 food_image 리스트를 가져올 수 있습니다.
(힌트 `_set`)

```suggestion
                'image'            : [image.image_url for image in food.....],
```